main/zoe/guide/contracts/escrow-to-vote.md

```diff
@@ -0,0 +1,15 @@
+# Escrow To Vote
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/test/unitTests/contracts/escrowToVote.js)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract implements coin voting. There are two roles: the
+Secretary, who can determine the question (a string), make voting
+invitations, and close the election; and the Voters, who can vote YES or
+NO on the question. The voters can only get the capability to vote
+by making an offer using a voter invitation and escrowing assets. The
+brand of assets is determined on contract instantiation through an
+issuerKeywordRecord. The creator gets the only Secretary
```

Comment:
```suggestion
An issuerKeywordRecord determines on contract instantiation the brand of the assets . The creator gets the only Secretary
```

Also remove the "The" at the end of the previous line if you do this.

Reply:
I think this one is less readable